src/app/api/comment/comment.service.ts

```diff
@@ -62,4 +62,22 @@ export class CommentService extends BaseService {
       },
     })
   }
+
+  async getCommentsByIds(commentIds: string[]) {
+    return await this.db.comment.findMany({
+      where: {
+        id: {
+          in: commentIds,
+        },
+      },
+    })
+  }
+
+  async getChildrenCommentByCommentId(commentId: string) {
```

Comment:
`getReplies`?

Reply:
Changed in 1970d59fa7dc97b69012e32efee7c852898dc439 commit